sass/directives/02_base_components/boxes/_boxes.scss

```diff
@@ -15,11 +15,18 @@
   }
 
   p {
-    color: $text-color-grey;
+    // color: $text-color-grey;
```

Comment:
Should we delete this rather than comment it out?

Reply:
yep!